### ISSUE TITLE
Question about distribute training

### ISSUE THREAD

**Issue by StepyHan** [melodyguan/enas#63]
I would like to know whether controller could collect multi rewards if I set child model sync_replicas=True, num_aggregate=2, num_replicas=2. I supposed that by using tf.train.SyncReplicasOptimizer, multi child models with different architecture would be trained to update shared parameter, but it seems only return one reward value. Hope for anwser，thanks ！
